src/acrn/acrn_driver.c

```diff
@@ -2557,6 +2557,19 @@ acrnInitPlatform(acrnPlatformInfoPtr pi, virNodeInfoPtr nodeInfo,
     return ret;
 }
 
+static
+void acrnNotifyLoadDomain(virDomainObjPtr domObj, int newDomain,
+	void *opaque ATTRIBUTE_UNUSED)
+{
+    if (newDomain) {
+        acrnDomainObjPrivatePtr priv;
+
+        /* populate hvUUID */
+        priv = domObj->privateData;
+        uuid_copy(priv->hvUUID, domObj->def->uuid);
```

Comment:
hvUUID is different from def->uuid and needs to be acquired via acrnAllocateVm(). Please refer to acrnDomainDefineXMLFlags() for its usage.

Reply:
The updated patch now is based on acrnAllocateVm().